src/annotator/notebook.js

```diff
@@ -6,8 +6,10 @@ import { createSidebarConfig } from './config/sidebar';
  *
  * @return {HTMLIFrameElement}
  */
-function createNotebookFrame(config) {
+function createNotebookFrame(config, groupId) {
```

Comment:
We're currently blindly passing the same config to the notebook that we pass to the sidebar, but much of it may irrelevant or simply not work at all in that context. I suggest that, separate to this PR, we should the config keys to be opt-in for the notebook rather than opt-out. When that is done, these two parameters could be merged into a single object.